packages/datasource/src/form/org/organization-sizes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "option1": "0-50人",
+  "ex4ab2q8x": "51-100人",
+  "sdpGA0bis": "100-200人",
+  "htO5CtXq7": "200-500人",
+  "JH0foSs6h": "500-1000人",
+  "Gfddv2G2w": "1000人以上"
+}
```

Comment:
I am not quite sure if these dropdown selections should be stored in the datasource package or just inside the `pages/account/organization/new` folder?

For me, the datasource is more suitable for the data which may be shared between repositories, like the Header & Footer's nav items. But for these dropdown data, they're more like one-time usage data, so it might be more straightforward if they're stored near their use cases.

Reply:
I think the organization profile page also needs this enums. We can keep this in `/pages/account/organization` or in `datasource`. 
We could make decision after all similar content are all developed.